server/data/dataAccess/db.js

```diff
@@ -1,9 +1,11 @@
 const logger = require('../../../log')
 const config = require('../../config')
 
-const { Pool } = require('pg')
+const { Pool, types } = require('pg')
 const fs = require('fs')
 
+types.setTypeParser(20, val => parseInt(val, 10))
+
```

Comment:
this seems to  be the recommended way to get pg to auto convert bigint (just booking Id currently) to an int.
I think we are save to do this and it means we don't have to have parseInts everywhere when comparing the db result